File: 520 - Detect Capital.java
class Solution {
    public boolean detectCapitalUse(String word) {
        int count = 0;
        for (char c : word.toCharArray()) {
            if (Character.isUpperCase(c)) {
                count++;
            }
        }
        boolean res = false;
        if (count == word.length() || count == 0 || (count == 1 && Character.isUpperCase(word.charAt(0)))) {
            res = true;
        }
        return res;
    }
}

File: 520 - Detect Capital.py
class Solution:
    def detectCapitalUse(self, word: str) -> bool:
        count = 0
        for char in word:
            if char.isupper():
                count += 1
        if len(word) == count or (count == 1 and word[0].isupper()) or count == 0:
            return True
        else:
            return False
            